Task_Position_the_Cubes/index.css

```diff
@@ -1,46 +1,58 @@
 /*  Start coding below for POSITION THE CUBES TASK*/
-.parent {
```

Comment:
The margin property should not have a percentage value. Use pixel values or other appropriate units.